api/chat_main_handler.go

```diff
@@ -366,6 +303,7 @@ func chat_stream(ctx context.Context, chatSession sqlc_queries.ChatSession, chat
 		// N:                n,
 		Stream: true,
 	}
+	ctx := context.Background()
 	stream, err := client.CreateChatCompletionStream(ctx, openai_req)
 	if err != nil {
 		RespondWithError(w, http.StatusInternalServerError, fmt.Sprintf("CompletionStream error: %v", err), nil)
```

Comment:
:robot: OpenAI

This patch removes the `ctx` argument from the `chat_stream` function and instead creates a new background context inside the function.

This change is an improvement because it simplifies the code and makes it more modular.

Please confirm that this change does not introduce any new functionality or bug risks before merging this patch.

Also, please update the comments on the pull request to reflect this change.

<!-- This is an auto-generated comment by OpenAI -->